packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -173,6 +173,10 @@ function defaultFocusRestorer(options: { originalElement?: HTMLElement | Window;
   const { originalElement, containsFocus } = options;
 
   if (originalElement && containsFocus && originalElement !== window) {
-    originalElement.focus();
+    // Make sure that the focus method actually exists
+    // In some cases the object might exist but not be a real element.
+    if (originalElement.focus) {
```

Comment:
nit: Collapse both if conditions into one.

Reply:
I want to keep them separate because this shouldn't need to exist once IE 11 is gone. I'll add that as a comment.